components/Navigation/index.js

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components";
+import Link from "next/link.js";
+
+export default function Navigation() {
+  return (
+    <StyledNavContainer>
+      <StyledNavList>
+        <li>
+          <StyledNavLink href="/">Home</StyledNavLink>
+        </li>
+        <li>
+          <StyledNavLink href="/bookmarks">Bookmarks</StyledNavLink>
+        </li>
+      </StyledNavList>
+    </StyledNavContainer>
+  );
+}
+
+const StyledNavContainer = styled.nav`
+  background-color: #7e6768;
+  position: fixed;
+  width: 100%;
+  bottom: 0;
+  padding: 1.5rem 0;
+  z-index: 999;
+  border-radius: 20px 20px 0 0;
+`;
+
+const StyledNavList = styled.nav`
```

Comment:
As the StyledNavContainer already is a nav element and you nest li elements inside the styledNavList, this should be an ul. 

```suggestion
const StyledNavList = styled.ul`
```